### PR TITLE
ansible@8: use resources for dependencies

### PR DESCRIPTION
**Pull request by cho-m**
This should help allow keeping `ansible@8` installs working while we drop support for `python@3.11` in Python bindings.

### CODE REVIEW

**Comment by chenrui333** [comment]
image

File: Formula/a/ansible@8.rb [hunk 1]
```diff
@@ -27,19 +28,19 @@ class AnsibleAT8 < Formula
   # `pkg-config` and `rust` are for bcrypt
   depends_on "pkg-config" => :build
   depends_on "rust" => :build
-  depends_on "certifi"
```
Comment (cho-m) [inline Formula/a/ansible@8.rb:30]: May actually keep this if get #193919 done.

**Review by p-linnane — APPROVED** [review]
Not crazy about the `certifi` change but it makes sense.

**Comment by cho-m** [comment]
> Not crazy about the `certifi` change but it makes sense.

I've been going back-and-forth on `certifi`. Specifically was experimenting with #193919 as an option.

**Pull request merged**